CRM/Core/Payment/PayPalImpl.php

```diff
@@ -903,6 +906,7 @@ public function doTransferCheckout(&$params, $component = 'contribute') {
 
     $cancelUrlString = "$cancel=1&cancel=1&qfKey={$params['qfKey']}";
     if (!empty($params['is_recur'])) {
+      // @todo use $this->getContributionID();
```

Comment:
Ok. Though possibly better just to add a class level todo as we can switch over various params to getters?